client/packages/host/src/Admin/LabelPrinterSettings.tsx

```diff
@@ -85,18 +90,30 @@ export const LabelPrinterSettings = () => {
         labelHeight: settings.labelHeight,
         labelWidth: settings.labelWidth,
         port: settings.port,
+        isUsb: settings.isUsb,
       });
     }
   }, [settings]);
 
   return (
     <>
       <SettingsSubHeading title={t('settings.label-printing')} />
+      <Setting
+        component={
+          <Switch
+            checked={draft.isUsb}
+            onChange={(_, checked) => onChange({ isUsb: checked, address: '' })}
+            color="primary"
+          />
+        }
+        title={t('settings.print-via-usb')}
+      />
```

Comment:
Had a chat with Mark and have concluded that we are to enable printing via USB through configuring it in LabelPrinterSettings (Settings -> Devices).

As a simple solution, I've extended LabelPrinterSettings node to include is_usb boolean. If true, it ignores the rest of the settings via the FE.

Not a set in stone solution but something to start with.